pkg/capabilities/consensus/ocr3/factory.go

```diff
@@ -3,14 +3,21 @@ package ocr3
 import (
 	"context"
 
+	"google.golang.org/protobuf/proto"
+
 	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/requests"
-
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/types"
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
 	"github.com/smartcontractkit/chainlink-common/pkg/services"
 )
 
+const (
+	defaultMaxPhaseOutputBytes = 1000000 // 1 MB
+	defaultMaxReportCount      = 20
```

Comment:
Can you move defaultBatchSize and defaultOutcomePruningThreshold here, for consistency?